registration/include/pcl/registration/impl/sample_consensus_prerejective.hpp

```diff
@@ -241,13 +240,30 @@ SampleConsensusPrerejective<PointSource, PointTarget, FeatureT>::computeTransfor
 
   // Feature correspondence cache
   std::vector<std::vector<int>> similar_features(input_->size());
+  std::vector<float> nn_distances(k_correspondences_);
```

Comment:
This gets cleared immediately at the top of the loop, so I don't think it needs to be initialized to a non-zero size here.
```suggestion
  std::vector<float> nn_distances;
```

Reply:
I think this should reserve the memory. By your changes will be skipped.

```suggestion
  std::vector<float> nn_distances;
  nn_distances.reserve(k_correspondences_);
```

Would be equal to before just, without first unnecessary allocation.

Reply:
Ah, I see.

Reply:
Done